sdk/Sdk.Generators/KnownTypes.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.Azure.Functions.Worker.Sdk.Generators
+{
+    // Trimmed version of https://github.com/dotnet/roslyn/blob/main/src/Features/Core/Portable/MakeMethodAsynchronous/AbstractMakeMethodAsynchronousCodeFixProvider.KnownTypes.cs
```

Comment:
Wondering if we can apply this pattern to some of the types we track in [this constants file](https://github.com/Azure/azure-functions-dotnet-worker/blob/main/sdk/Sdk.Generators/Constants.cs).

Reply:
Yea, I think we should. Let's address that in a follow up PR.